src/compas_fab/ghpython/components/ros_robot.py

```diff
@@ -6,6 +6,7 @@
 import compas
 from compas_fab.ghpython.components import create_id
 from compas_fab.ghpython.components.icons import ros_robot_icon
+from compas_fab.ghpython import RobotArtist
```

Comment:
This is still possible, but it's deprecated.  Should be 
```suggestion
from compas_ghpython.artists import RobotModelArtist
```